doc/index.rst

```diff
@@ -6,10 +6,45 @@
 Welcome to Sphinx-Gallery's documentation!
 ==========================================
 
-Sphinx extension for automatic generation of an example gallery.
+A Sphinx extension that builds an HTML version of any python
+script and puts it into an examples gallery.
```

Comment:
Suggested rephrasing:

A sphinx (link to sphinx homepage)  extension that builds an HTML gallery of examples from any set of Python scripts.